### ISSUE TITLE
Bot cannot log in due to api.oauth2.applications not being a function

### ISSUE THREAD

**Issue by vladfrangu** [UnseenFaith/komada#299]
### Version of Komada
0.20.4

### Are core files modified?
No.

### Describe the problem
At logon using a Bot Token, the error `TypeError: this.api.oauth2.applications` is thrown
[See here for the full error](https://hastebin.com/goguhajaye.tex)

### Expected Behaviour
Bot logs in

### Actual Behaviour
Bot doesn't work at all.

### Steps to Reproduce.
Use latest indev Komada

**Comment by pedall** [comment]
@kyra what is the 0.20.3 commit hash we could use until this is fixed?

**Comment by UnseenFaith** [comment]
@KingDGrizzle @pedall This issue should be fixed by #300
I tested it on my own OAuth bot and it seemed to log in fine.

**Comment by pedall** [comment]
thank you very much Faith. Are you able to guess when this PR will be merged into indev? (to see if its worth to reinstall komada with a commit lock for now)

**Comment by UnseenFaith** [comment]
Just waiting on @kyranet or @CyberiumShadow to take a look at it.

You can go ahead and install it from my repository though if you want to test it.

`npm install UnseenFaith/komada#indev`

**Comment by CyberiumShadow** [comment]
Fixed in #300 feel free to move back to the dirigeants indev branch
